frontend/src/typings/react-table-config.d.ts

```diff
@@ -0,0 +1,132 @@
+// Since usePagination and useSortBy are considered plugins, it is necessary to declare the types explicitly.
```

Comment:
This means the types are global even though different instances of `react-table` may not use the above plugins. Suggest creating explicit interfaces that extend the plugins we need instead of a `.d.ts` file:

```ts
interface FeedbackTableOptions<D extends Record<string, unknown>>
  extends UsePaginationOptions<D>,
    UseSortByOptions<D> {}
```

and use that in useTable calls:

```tsx
const { prepareRow, headerGroups, page, gotoPage } = useTable<Data>(
    {
      columns,
      data,
      initialState: { pageIndex: currentPage, pageSize: 9 },
    } as FeedbackTableOptions,
    useSortBy,
    usePagination
)
```